### PR TITLE
minor fixes on the Transfers API, Sidecar, and Polkadot.js API pages

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR just addresses some typos, grammar issues, and formatting

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira

### After Translation Requirements

- [x] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/build/substrate-api/sidecar.md [hunk 1]
```diff
@@ -34,62 +34,66 @@ npm install @substrate/api-sidecar@{{ networks.moonbase.substrate_api_sidecar.st
...
+If the installation and configuration are successful, you should see this output in the console:
 
 ![Successful Output](/images/builders/build/substrate-api/sidecar/sidecar-1.png)
 
-## Substrate API Sidecar Endpoints {: #substrate-api-sidecar-endpoints } 
+## Substrate API Sidecar Endpoints {: #substrate-api-sidecar-endpoints }
 
 Some of the commonly used Substrate API Sidecar endpoints include:
 
  - **GET /blocks​/head** — Get the most recently finalized block. The optional parameter `finalized` can be set to `false` to the get the newest known block, which may not be finalized.
```
Comment (albertov19) [inline builders/build/substrate-api/sidecar.md:95]:
```suggestion
 - **GET /blocks​/head** — Get the most recently finalized block. The optional parameter `finalized` can be set to `false` to the get the newest known block, which may not be finalized
```

File: builders/build/substrate-api/sidecar.md [hunk 2]
```diff
@@ -34,62 +34,66 @@ npm install @substrate/api-sidecar@{{ networks.moonbase.substrate_api_sidecar.st
...
- - **GET /blocks/head/header** — Get the most recently finalized block header. The optional parameter `finalized` can be set to `false` to the get the newest known block header, which may not be finalized. 
+ - **GET /blocks/head/header** — Get the most recently finalized block header. The optional parameter `finalized` can be set to `false` to the get the newest known block header, which may not be finalized.
```
Comment (albertov19) [inline builders/build/substrate-api/sidecar.md:96]:
```suggestion
 - **GET /blocks/head/header** — Get the most recently finalized block header. The optional parameter `finalized` can be set to `false` to the get the newest known block header, which may not be finalized
```

File: builders/build/substrate-api/sidecar.md [hunk 3]
```diff
@@ -34,62 +34,66 @@ npm install @substrate/api-sidecar@{{ networks.moonbase.substrate_api_sidecar.st
...
  - **GET /blocks/{blockId}** — Get a block by its height or hash.
```
Comment (albertov19) [inline builders/build/substrate-api/sidecar.md:97]:
```suggestion
 - **GET /blocks/{blockId}** — Get a block by its height or hash
```

File: builders/build/substrate-api/sidecar.md [hunk 4]
```diff
@@ -34,62 +34,66 @@ npm install @substrate/api-sidecar@{{ networks.moonbase.substrate_api_sidecar.st
...
  - **GET /accounts/{accountId}/balance-info** — Get balance information for an account.
```
Comment (albertov19) [inline builders/build/substrate-api/sidecar.md:98]:
```suggestion
 - **GET /accounts/{accountId}/balance-info** — Get balance information for an account
```

**Review by albertov19 — APPROVED** [review]
LGTM

**Comment by eshaben** [comment]
Good catches! Thanks!!

**Pull request merged**